.vscode/settings.json

```diff
@@ -21,5 +21,9 @@
     "Renci",
     "Swashbuckle"
   ],
-  "java.configuration.updateBuildConfiguration": "interactive"
+  "java.configuration.updateBuildConfiguration": "interactive",
+  "editor.codeActionsOnSave": {
```

Comment:
Added this to top VSCode performing it's own organize imports which is not compatible with what we have configured for TNO.  Some times I hate the linters.